apis/pkg/v1/interfaces.go

```diff
@@ -18,12 +18,19 @@ package v1
 
 import (
 	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 
 	xpv1 "github.com/crossplane/crossplane-runtime/apis/common/v1"
 	"github.com/crossplane/crossplane-runtime/pkg/resource"
 )
 
+const (
+	// ParentLabel is used as key for the owner package label we add to the
+	// revisions. Its corresponding value should be the name of the owner package.
+	ParentLabel = "pkg.crossplane.io/package"
```

Comment:
Nit: I believe our typical convention with these constants is to name them something like `LabelFoo` or `LabelKeyFoo` where `Foo` matches the actual label content. So this would be something like `LabelKeyPackage`. The rationale is that if we add more labels we can group them all together and they'll all start with `Label`.